scripts/backup.sh

```
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+R='\033[0;31m'
+G='\033[0;32m'
+Y='\033[0;33m'
+B='\033[0;34m'
+DEF='\033[0m'
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+ARCHIVE_NAME="shutter-api-keyper-$(date +%Y-%m-%dT%H-%M-%S).tar.xz"
+
+source "${SCRIPT_DIR}/../.env"
+
+WORKDIR=$(mktemp -d)
+
+cleanup() {
+  rv=$?
+  set +e
+  echo -e "${R}Unexpected error, exit code: $rv, cleaning up.${DEF}"
+  docker compose unpause || true
+  exit $rv
+}
+
+trap cleanup EXIT
+
+echo -e "${G}Creating backup archive${DEF}"
+
+mkdir -p "${SCRIPT_DIR}/../data/backups"
+
+echo -e "${B}[1/7] Pausing all services except database...${DEF}"
+docker compose pause keyper || true
```

this swallows errors, shouldn't we abort in that case? Same for the other pause commands below and for the down command in the recovery script.